agent_id/src/test/AgentIDTests.cpp

```diff
@@ -31,6 +31,31 @@ TEST(AgentID, ConstructionOfHugeID) {
     delete id1;
 }
 
+TEST(AgentID, ConstructionOfHugeStringBasedID) {
+
+    essentials::AgentID* id1;
+    std::string agent_id_param = "instjqndaczdtsshocazkdtgpxknutnrkc79bbd8f95j4wv4";
```

Comment:
`const std::string` To let the reader know that this value does not change (in this case its obvious but its a good programming guideline)